src/components/Form/Form.jsx

```diff
@@ -32,26 +31,35 @@ const Form = ({
 
   const submitHandler = (event) => {
     event.preventDefault();
-    let isFormValid = Object.values(errors).every(
-      (error) => error.length === 0
-    );
 
-    Object.entries(customerInfo).forEach(
-      ([inputName, inputValue]) =>
-        inputValue.length === 0 &&
-        setErrors((previous) => ({
-          ...previous,
+    let inputsErrors = {};
+
+    Object.entries(customerInfo).forEach(([inputName, inputValue]) => {
+      if (inputValue.length === 0) {
+        inputsErrors = {
+          ...inputsErrors,
           [inputName]: inputName + " can`t be blank",
-        }))
+        };
+      }
+    });
+
+    let isFormValid = Object.values(inputsErrors).every(
+      (error) => error.length === 0
```

Comment:
What does it mean `error.length === 0`?

Reply:
to check inputs errors if input error is empty its mear its valid